sys/kern/subr_bus.c

```diff
@@ -5156,7 +5154,7 @@ static moduledata_t root_bus_mod = {
 	root_bus_module_handler,
 	NULL
 };
-DECLARE_MODULE(rootbus, root_bus_mod, SI_SUB_DRIVERS, SI_ORDER_FIRST);
+DECLARE_MODULE(rootbus, root_bus_mod, SI_SUB_INTR, SI_ORDER_FIRST);
```

Comment:
this is premature at this time. I'm not going to land it just yet.

Reply:
Fine by me.  Soon to be later in the series.